InternalDocs/vm-state.md

```diff
@@ -5,22 +5,12 @@
 - **Tier 1** is the classic Python bytecode interpreter.
   This includes the specializing adaptive interpreter described in [PEP 659](https://peps.python.org/pep-0659/) and introduced in Python 3.11.
 - **Tier 2**, also known as the micro-instruction ("uop") interpreter, is a new interpreter with a different instruction format.
-  It will be introduced in Python 3.13, and also forms the basis for a JIT using copy-and-patch technology that is likely to be introduced at the same time (but, unlike the Tier 2 interpreter, hasn't landed in the main branch yet).
+  It was introduced in Python 3.13, and also forms the basis for a JIT using copy-and-patch technology. See [Tier 2](tier2_engine.md) for more information.
 
 # Frame state
 
 Almost all interpreter state is nominally stored in the frame structure.
-A pointer to the current frame is held in `frame`. It contains:
-
-- **local variables** (a.k.a. "fast locals")
```

Comment:
There is a reference below to 'fast locals' which now is out of context. (I can't comment on that line because it has not changed).